File: eRecipes/eRecipes.Service/Database/Izvjestaj.cs
using System;
using System.Collections.Generic;

namespace eRecipes.Service.Database
{
    public  class Izvjestaj
    {
      public int IzvjestajId { get; set; }
      public int ReceptId { get; set; }
      public int BrojLajkova { get; set; } = 0;
       public int BrojOmiljenih { get; set; } = 0;
      public DateTime DatumIzvjestaja { get; set; }
        public virtual Recept Recept { get; set; } = null!;
}
}

File: eRecipes/UI/erecipes_desktop/lib/models/izvjestaj.dart
import 'package:erecipes_desktop/models/recept.dart';
import 'package:json_annotation/json_annotation.dart';

part 'izvjestaj.g.dart';

@JsonSerializable()
class Izvjestaj {
  int? izvjestajId;
  int? receptId;
  int? brojLajkova;
  int? brojOmiljenih;
  DateTime? datumIzvjestaja;
  Recept? recept;

  Izvjestaj({required this.receptId});

  factory Izvjestaj.FromJson(Map<String, dynamic> json) =>
      _$IzvjestajFromJson(json);

  Map<String, dynamic> toJson() => _$IzvjestajToJson(this);
}
